educa/tests/courses/test_views.py

```diff
@@ -48,3 +48,23 @@ def test_module_content_list_template_used(self):
         self.assertTemplateUsed(
             response, "courses/manage/module/content_list.html"
         )
+
+    def test_text_content_get_stored_in_module_contents(self):
+        text_content = self.create_text()
+        module_content = self.create_module_content(content=text_content)
+        self.assertEqual(text_content, module_content.item)
+
+    def test_image_content_get_stored_in_module_contents(self):
+        image_content = self.create_image()
+        module_content = self.create_module_content(content=image_content)
+        self.assertEqual(image_content, module_content.item)
+
+    def test_video_content_get_stored_in_module_contents(self):
+        video_content = self.create_video_url()
+        module_content = self.create_module_content(content=video_content)
+        self.assertEqual(video_content, module_content.item)
+
+    def test_file_content_get_stored_in_module_contents(self):
+        file_content = self.create_file()
+        module_content = self.create_module_content(content=file_content)
+        self.assertEqual(file_content, module_content.item)

```

Comment:
This is just testing the test mixing which we created. Ideally we should test the functionality added in the PR. In this case it's the view which needs testing.